src/types.ts

```diff
@@ -124,6 +124,7 @@ export enum MarkupInputType {
   Number = 'number',
   Boolean = 'boolean',
   String = 'string',
+  Map = 'map',
```

Comment:
Could you add test for Argument with a `map` in `getMarkup`, `getMir` and `getJs`?